lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -1243,6 +1243,7 @@ namespace Js
         if(scriptContext->GetThreadContext()->IsRuntimeInTTDMode())
         {
             AddFunctionToLibraryObjectWithPropertyName(globalObject, _u("telemetryLog"), &GlobalObject::EntryInfo::TelemetryLog, 3);
+            AddFunctionToLibraryObjectWithPropertyName(globalObject, _u("emitTTDLog"), &GlobalObject::EntryInfo::EmitTTDLog, 2);
         }
```

Comment:
Personally I don't like the idea of polluting globalObject with non-spec'd properties without user being aware of it. I had this concern initially as well and now we are adding another property.
If we can't do without having all this utility methods then we should have a JsRT API like JsTTDEnableObjectOnGlobal(const char *ttdName, size_t length); when this is called we will add a object (named ttdName) on global object with all private methods like telemetryLog, emitTTDLog etc.